packages/esm-patient-list-app/src/AddPatientToList/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import Search from 'carbon-components-react/lib/components/Search';
+import Button from 'carbon-components-react/lib/components/Button';
+import Checkbox from 'carbon-components-react/lib/components/Checkbox';
+import { usePatientListData } from '../patientListData';
+import { useTranslation } from 'react-i18next';
+import styles from './add-patient-to-list.scss';
+
+import { OpenmrsCohort, addPatientToList } from '../patientListData/api';
+import SkeletonText from 'carbon-components-react/es/components/SkeletonText';
+import { toOmrsIsoString } from '@openmrs/esm-framework';
+
+const CheckboxedPatientList = ({ name, uuid, checked, handleChange }) => {
+  return (
+    <div className={styles.checkbox}>
+      <Checkbox checked={checked} labelText={name} id={uuid} onChange={(e) => handleChange(uuid, e)} />
+    </div>
+  );
+};
+
+const AddPatient: React.FC<{ close: () => void; patientUuid: string }> = ({ close, patientUuid }) => {
+  const { t } = useTranslation();
+  const [searchValue, setSearchValue] = React.useState('');
+  const { loading, data } = usePatientListData(undefined, undefined, undefined, searchValue);
+  const [selectedLists, setSelectedList] = useState({});
```

Comment:
`useState` vs `React.useState`? Why not use one consistently?

Also what's the type of this state?